agent/roles/manifests/templates/install-config_baremetal_yaml.j2

```diff
@@ -65,10 +65,22 @@ platform:
       - {{ ingress_vip }}
 {% endfor %}
 {% if agent_install_config_bm_hosts == "true" %}
+{% set bmc_addresses = agent_nodes_bmc_addresses.split(',') %}
+{% set bmc_passwords = agent_nodes_bmc_passwords.split(',') %}
+{% set bmc_usernames = agent_nodes_bmc_usernames.split(',') %}
+    provisioningHostIP: {{ cluster_provisioning_ip }}
+    provisioningNetworkInterface: {{ cluster_provisioning_interface }}
+    provisioningNetworkCIDR: {{ provisioning_network }}
+    provisioningNetwork: {{ provisioning_network_profile }}
     hosts:
 {% for hostname in hostnames %}
     - name: {{ hostname }}
       bootMACAddress: {{ macs[loop.index0] }}
+      bmc:
+        address: {{ bmc_addresses[loop.index0] }}
+        username: {{ bmc_usernames[loop.index0] }}
+        password: {{ bmc_passwords[loop.index0] }}
+        disableCertificateVerification: false
```

Comment:
Please feel free to ignore this comment.
I noticed that `disableCertificateVerification` is hard coded here, but is configurable in the `assisted-service` https://github.com/openshift/assisted-service/blob/25d0ae75d6a4a268f7555339a74cc6b050ba23aa/internal/installcfg/installcfg.go#L21

Should it also be configurable here?

Reply:
Thanks @ahussey-redhat. So this setting doesn't affect the installation or the resulting baremetalhosts, and making it configurable would just require a new environment variable which wouldn't be used in any CI tests so I don't think we need it to be configurable.